deploy/sdk/src/dynamo/sdk/cli/containerize.py

```
@@ -0,0 +1,171 @@
+    tag: t.Optional[str] = typer.Option(
+        None, "--tag", "-t", help="Docker image tag (defaults to generated tag)"
+    ),
+) -> None:
+    """
+    Containerize a Dynamo service directly into a Docker image.
+
+    This command combines the build and containerization steps, creating a
+    Dynamo package and immediately building it into a Docker container.
+    """
+    try:
+        from dynamo.sdk.cli.build import BuildConfig, Package
```

Pylint (3.3.7)

[error] Unable to import 'dynamo.sdk.cli.build'

(E0401)